layouts/default.vue

```diff
@@ -1,6 +1,12 @@
 <template>
   <div>
     <PageHeader />
+    <a
```

Comment:
If we have such a link, then it should porbably be the very first tabbable element on the page, before even anything in the header. Also, there is no element with ID "main".

For now, please remove this and we can introduce it, refined, in a later iteration.